packages/ckeditor5-enter/tests/enterobserver.js

```diff
@@ -84,6 +86,32 @@ describe( 'EnterObserver', () => {
 		sinon.assert.notCalled( enterSpy );
 	} );
 
+	// See https://github.com/ckeditor/ckeditor5/issues/13321.
+	it( 'should handle the insertParagraph input type and fire the soft enter event if shift key is pressed in Safari', () => {
```

Comment:
I'd also test the following scenario:

1.  A user presses `shift`.
2.  A user releases `shift`.
3.  A user presses `enter`.

Reply:
Added test.